x-pack/plugins/security_solution/public/common/components/events_viewer/default_event_headers.tsx

```diff
@@ -29,6 +29,10 @@ export const defaultHeaders: ColumnHeaderOptions[] = [
     columnHeaderType: defaultColumnHeaderType,
     id: 'event.module',
   },
+  {
+    columnHeaderType: defaultColumnHeaderType,
+    id: 'agent.type',
```

Comment:
- While desk testing, `agent.type` appears as a default column in the `Hosts`, `Network`, and `Users` Events tables

- Per our offline discussion, column customizations don't appear to be persisted to local storage for the `Users` Events table, but that's also true in `8.3.2`

Reply:
In `8.3.2`, the `Hosts > External alerts` table has a different set of default columns:
- `Actions`
- `@timestamp`
- `event.module`
- `event.dataset`
- `event.category` // only in Hosts > External alerts
- `event.severity` // only in Hosts > External alerts
- `observer.name` // only in Hosts > External alerts
- `host.name`
- `message`
- `agent.id` // only in Hosts > External alerts
- `agent.type` // only in Hosts > External alerts

than the `Hosts > Events` table, which in `8.3.2` is:
- `Actions`
- `@timestamp`
- `message`
- `host.name`
- `event.module`
- `event.dataset`
- `event.action`
- `user.name`
- `source.ip`
- `destination.ip`

In the PR branch, the `Hosts > Events` table has the same set of columns has the `8.3.2` version of the table, with the addition of `agent.type`.

Would you be willing to confirm that going forward, the intended design is for the following columns to be displayed, regardless of whether the `Show only external alerts` checkbox is checked?

- `Actions`
- `@timestamp`
- `message`
- `host.name`
- `event.module`
- `agent.type`
- `event.dataset`
- `event.action`
- `user.name`
- `source.ip`
- `destination.ip`

Reply:
Hey Andrew, that's correct, we are going with the events table default columns, with the addition of agent.type. I will add this note to the description.